membership-attribute-service/app/controllers/AccountController.scala

```diff
@@ -299,16 +299,17 @@ class AccountController(
       currency: Currency,
       billingPeriod: RecurringPeriod,
       nextPaymentDate: LocalDate,
-  ) = SimpleEitherT.right(sendEmail(updateAmountEmail(email, contact, newPrice, currency, billingPeriod, nextPaymentDate)))
+  )(implicit logPrefix: LogPrefix) =
+    SimpleEitherT.right(sendEmail.send(updateAmountEmail(email, contact, newPrice, currency, billingPeriod, nextPaymentDate)))
 
   private def sendSubscriptionCancelledEmail(
       email: String,
       contact: Contact,
       plan: SubscriptionPlan.AnyPlan,
       cancellationEffectiveDate: Option[LocalDate],
-  ) =
+  )(implicit logPrefix: LogPrefix) =
     SimpleEitherT
-      .right(sendEmail(subscriptionCancelledEmail(email, contact, plan, cancellationEffectiveDate)))
+      .right(sendEmail.send(subscriptionCancelledEmail(email, contact, plan, cancellationEffectiveDate)))
```

Comment:
this used to use an `apply` method, so I updated it to use a normal method as it's less confusing